src/test/ui/asm/naked-functions-testattrs.stderr

```diff
@@ -0,0 +1,27 @@
+error[E0788]: cannot use testing attributes with `#[naked]`
+  --> $DIR/naked-functions-testattrs.rs:12:1
+   |
+LL | #[naked]
+   | ^^^^^^^^
```

Comment:
In the output, can we also point with secondary spans to the other non applicable attributes?